src/WebJobs.Script/Binding/Http/HttpTriggerAttributeBindingProvider.cs

```diff
@@ -252,6 +256,22 @@ internal static void ApplyBindingData(object target, IDictionary<string, object>
                     }
                 }
 
+                {
+                    string auth = null;
+
+                    if (request.Headers.Authorization != null)
+                    {
+                        auth = request.Headers.Authorization.Parameter;
+                    }
+                    else
+                    {
+                        // The (optional) id token from easy auth.
+                        // This can be used in a token exchange.
+                        auth = request.GetHeaderValueOrDefault("X-MS-TOKEN-AAD-ID-TOKEN"); // JWT
```

Comment:
if `Parameter` is null above should we try to pull this header value?  Also, move to script constants.